features/mesh/dataplanes/DataplaneDetailsStandardProxy.feature

```diff
@@ -0,0 +1,155 @@
+
+  Scenario: Overview tab has expected content
+    Then the page title contains "dpp-1-name-of-dataplane"
+    And the "$detail-view" element contains "dpp-1-name-of-dataplane"
+    And the "$warnings" element doesn't exist
+    And the "$details" element contains "online"
+    And the "$inbounds" element contains "healthy"
+    And the "$inbounds" element contains "193.107.134.106:1328"
+    And the "$inbounds" element contains "44.167.201.218:62098"
+    And the "$inbounds" element contains "kuma.io/protocol:http"
+    And the "$inbounds" element contains "kuma.io/zone:zone-1"
+    And the "$subscriptions" element contains "Connected: Feb 17, 2021, 7:33 AM"
+    And the "$subscriptions" element contains "CP instance ID: dpp-1-cp-instance-id"
+
+    When I click the ".accordion-item:nth-child(1) [data-testid='accordion-item-button']" element
+
+    Then the "$status-cds" element contains "CDS"
+    And the "$status-cds" element contains "1"
+    And the "$status-cds" element contains "2"
+    And the "$status-eds" element contains "EDS"
+    And the "$status-eds" element contains "3"
+    And the "$status-eds" element contains "4"
+    And the "$status-lds" element contains "LDS"
+    And the "$status-lds" element contains "6"
+    And the "$status-rds" element contains "RDS"
+    And the "$status-rds" element contains "0"
+
+  Scenario: Policies tab has expected content
+    Given the URL "/meshes/default/dataplanes/dpp-1-name-of-dataplane/policies" responds with
+      """
+      body:
+        items:
+          - matchedPolicies:
+              FaultInjection:
+                - sources:
+                   - match:
+                      kuma.io/service: service-a
+      """
+    And the URL "/meshes/default/dataplanes/dpp-1-name-of-dataplane/rules" responds with
+      """
+      body:
+        items:
+          - name: ''
+          - name: demo-app_kuma-demo_svc_5000
+            service: demo-app_kuma-demo_svc_5000
+            tags:
+              kuma.io/service: demo-app_kuma-demo_svc_5000
+      """
+
+    When I click the "$policies-tab" element
+    And I click the "$policy-item-button" element
+
+    Then the "$policies-view" element contains "kuma.io/service:service-a"
+
+    When I click the "$rule-item-button" element
+
+    Then the "$policies-view" element contains "kuma.io/service:demo-app_kuma-demo_svc_5000"
+
+  Scenario: Clusters tab has expected content
+    Given the URL "/meshes/default/dataplanes/dpp-1-name-of-dataplane/clusters" responds with
+      """
+      body:
+        access_log_sink::observability_name::access_log_sink
+        access_log_sink::default_priority::max_connections::1024
+        access_log_sink::default_priority::max_pending_requests::1024
+        access_log_sink::default_priority::max_requests::1024
+      """
+
+    When I click the "$clusters-tab" element
+
+    Then the "$clusters-view" element contains "access_log_sink::observability_name::access_log_sink"
```

Comment:
They’re _quite_ similar, but not completely:

- Standard DPPs have inbounds; Gateways don’t (they have a gateway property instead)
- Standard DPPs **and** Delegated Gateways share the same Policies tab (i.e. the one with Policies and Rules); Built-in Gateways use a completely different Policies tab

Basically, we have some overlap between the three types but also differences that create three sensible test categories.

Reply:
I'm just wondering about whether we should move this test to be about `policies` rather than `dataplanes` and then make a table test, so we don't duplicate the same test in two different files. I might be wrong, but did I see that the only thing that differs between the two tests is the word `builtin/delegated` in the URL and `BUILTIN/DELEGATED` in the mock?

Anyway, not a biggie so not a blocker if you don't like that idea. (and maybe I saw wrong that those two things where the only differences 🤷 )

Reply:
I’m going to reconsider this test structure outside of this PR. I’m not totally sure about it either. I think on the scale from different to similar, this falls somewhere in the middle and makes it harder to decide whether one test can cover multiple types or not.

Let me think about this some more (outside of this PR).